go-controller/pkg/ovn/policy_test.go

```diff
@@ -1753,6 +1757,7 @@ var _ = ginkgo.Describe("OVN NetworkPolicy Operations", func() {
 				// TODO: test server does not garbage collect ACLs, so we just expect policy & deny portgroups to be removed
 				acls := []libovsdb.TestData{}
 				acls = append(acls, gressPolicy1ExpectedData[:len(gressPolicy1ExpectedData)-1]...)
+				acls = append(acls, defaultDenyExpectedData[:len(defaultDenyExpectedData)-2]...)
```

Comment:
hmm for here and above changes in the unit tests do we really need this new line? can't we update the previous line to be `-2` ?

Reply:
`gressPolicy1ExpectedData` contains db objs for netpol-specific port group, and `defaultDenyExpectedData` contains db objs for shared namespace default deny port group. 
-2 if to drop 2 default deny port groups that will be deleted explicitly